app/views/response/response.html.erb

```diff
@@ -86,6 +87,21 @@ jQuery(document).ready(function() {
     <%= hidden_field_tag('return', @return) %>
     <%= hidden_field_tag  "isSubmit", :id=>"isSubmit" %>
 <% end %>
+
```

Comment:
We have updated the code so that the link is now shown in the 'others work' page next to the begin, update or edit